tool/gravity/cli/clusterupdate.go

```diff
@@ -60,6 +60,9 @@ func updateTrigger(
 	updatePackage string,
 	manual, noValidateVersion bool,
 ) error {
+	if err := updateRotateCertificates(localEnv); err != nil {
```

Comment:
Just curious, any reason this is not done as a part of the "init" upgrade phase?

Reply:
No reason - I did not think about it, will move it there.